pages/pipelines/trigger_step.md.erb

```diff
@@ -21,10 +21,10 @@ All builds created by a trigger step will have the same author as the parent bui
 * be a member of your organization
 * have a verified email address
 
-If you have [Teams](/docs/pipelines/permissions) enabled in your organization, one of the following conditions must be met:
+If you have [Teams](/docs/pipelines/permissions) enabled in your organization, *one* of the following conditions must be met:
 
 * The authoring user must have 'Build' permission on *every* pipeline that will be triggered
-* The source pipeline and the target pipeline must share a team that can 'Build'
+* The triggering build has no owner and no unblocker, *and* the source pipeline and the target pipeline share a team that can 'Build'
```

Comment:
Is "owner" a new concept? Or is it the same as "creator"?

Reply:
(and I'll look into that in more depth and make sure our concepts are lined up correctly when it isn't 23:25 :+1: )